common/operations.go

```diff
@@ -21,47 +22,39 @@ const (
 
 var (
 	Operations = adapter.Operations{
-		InstallBookInfoCommand: &adapter.Operation{
-			Type: int32(meshes.OpCategory_SAMPLE_APPLICATION),
-			Properties: map[string]string{
-				adapter.OperationDescriptionKey:  "Istio Book Info Application",
-				adapter.OperationVersionKey:      "",
-				adapter.OperationTemplateNameKey: "templates/bookinfo.yaml",
-				adapter.OperationServiceNameKey:  "productpage",
```

Comment:
Why remove the service name?

Reply:
@mgfeller I was thinking of having that as a part of extending the operations struct, We would definitely need service name. :)

Reply:
@kumarabd ok thanks :+1: I'm afraid I don't quite understand the reasoning behind whether a field is included here or not, and what kind of flexibility you're thinking about (in comment about `Template`) .